contracts/fixed_price.scilla

```diff
@@ -788,6 +802,8 @@ procedure DoSetOrder(order: OrderParam)
           (* create a buy order *)
           match is_native_zil with 
           | False =>
+            ZRC2TransferFrom payment_token_address _sender _this_address sale_price
```

Comment:
@razill it is the responsibility of the external application to call the `IncreaseAllowance` to set the marketplace as the operator, right?

Reply:
Yes @inamirandaz In order for a seller to sell on the marketplace with ZRC2, the user has to `IncreaseAllowance` to the marketplace contract